File: LeetCode/Easy/0832-flipping-an-image/0832-flipping-an-image.cpp
class Solution {
public:
    vector<vector<int>> flipAndInvertImage(vector<vector<int>>& image) {
        int n = image.size();
        vector<vector<int>> ans (n, (vector<int> (n,0)));
        for(int i=0; i<image.size(); i++){
            for(int j=0; j<image[0].size(); j++){
                ans[i][n-j-1] = 1 - image[i][j];
            }
        }
        return ans;
    }
};

File: LeetCode/Easy/0832-flipping-an-image/0832-flipping-an-image.py
class Solution:
    def flipAndInvertImage(self, image: List[List[int]]) -> List[List[int]]:
        n = len(image)
        result = []
        for i in range(n):
            tmp = []
            for j in range(n):
                tmp.append(1 - image[i][n-j-1])
            result.append(tmp)
        return result